src/t8_geometry/t8_geometry_implementations/t8_geometry_linear.cxx

```diff
@@ -54,6 +56,356 @@ t8_geometry_linear::t8_geom_evaluate_jacobian (t8_cmesh_t cmesh, t8_gloidx_t gtr
   SC_ABORT ("Not implemented.");
 }
 
+/** Check if a point lies inside a vertex
+ * 
+ * \param[in] vertex_coords The coordinates of the vertex
+ * \param[in] point         The coordinates of the point to check
+ * \param[in] tolerance     A double > 0 defining the tolerance
+ * \return                  0 if the point is outside, 1 otherwise.  
+ */
+static int
+t8_vertex_point_inside (const double vertex_coords[3], const double point[3], const double tolerance)
```

Comment:
Maybe we can move these function to the geometry helpers file instead, since other geometries could use them as well

Reply:
Done